Maskbook/Scene/App/FIleService/View/FileServiceView.swift

```diff
@@ -164,6 +169,20 @@ final class FileServiceOnboardViewModel: ObservableObject {
             .union(items)
             .map { $0 }
     }
+    
+    func addFromLocalfiles() {
+        coordinator.present(scene: .fileServiceLocalFileSource(selectFileHandler: selectFileHandler), transition: .panModel(animated: true))
+    }
+}
+
+extension FileServiceOnboardViewModel: FileServiceSelectFileDelegate {
```

Comment:
Maybe `FileServiceSelectFileDelegate` only need to pass the `Data` of the file to `FileServiceOnboardViewModel` to make the logic simpler. Abstracting the actions, like converting image to Data, converting file url to Data and compressing data, into separate objects will be more clear and maintainable.

Reply:
`Data ` may not enough, file name and file type is needed. I can pass fileURL to make it more clear and maintainable

Reply:
Then pass both `Data` and `FileType`, `Data` processing like converting and compressing should be done before pass the result to `FileServiceOnboardViewModel`

Reply:
ok